### ISSUE TITLE
Ability to set default language

### ISSUE THREAD

**Issue by MeesJ** [sbs20/scanservjs#612]
Right now it looks like scanservjs adjusts the language of the web ui to the language the client's device is set to. It'd be nice to have the option to set a language (for example via a Docker environmental variable) for all clients, regardless of their device's language.

**Comment by ceskyDJ** [comment]
It has already been added (see #583). More information are in https://github.com/sbs20/scanservjs/commit/71904e23001406213f866314d190214f537ac1c0.

**Comment by MeesJ** [comment]
That's right, but that requires users to set a query string. It would be much more to set the locale for everyone via a setting such as an environmental variable.

**Comment by ceskyDJ** [comment]
Ok, you're right. I think there should be way to set up in standard config of Scanservjs. I thought there is a way, but it isn't. For my application, query string is just fine solution but I realize that explicit configuration would be even better.
